### PR TITLE
PMM-2045 MySQL Group Replication Dashboard

### PR DESCRIPTION
**Pull request by adivinho**
- [ ] https://github.com/Percona-Lab/pmm-submodules/pull/1061

### CODE REVIEW

File: queries-mysqld-group-replication.yml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+mysql_perf_schema_replication_group:
+  query: "SELECT CHANNEL_NAME as channel_name, MEMBER_ID as member_id, MEMBER_HOST as member_host, MEMBER_PORT as member_port, MEMBER_STATE as member_state, MEMBER_ROLE as member_role, MEMBER_VERSION as member_version, CASE WHEN MEMBER_STATE = 'ONLINE' THEN 1 WHEN MEMBER_STATE = 'RECOVERING' THEN 2 WHEN MEMBER_STATE = 'OFFLINE' THEN 3 WHEN MEMBER_STATE = 'ERROR' THEN 4 WHEN MEMBER_STATE = 'UNREACHABLE' THEN 5 END as member_info FROM performance_schema.replication_group_members WHERE MEMBER_ID=@@server_uuid"
+  metrics:
+    - channel_name:
+        usage: "LABEL"
+        description: "Collect metrics from performance_schema.replication_group_members"
```
Comment (BupycHuk) [inline queries-mysqld-group-replication.yml:6]: Why all descriptions are the same?

File: queries-mysqld-group-replication.yml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+LAST_QUEUED_TRANSACTION_START_QUEUE_TIMESTAMP - LAST_QUEUED_TRANSACTION_ORIGINAL_COMMIT_TIMESTAMP 'transport_time_seconds', 
+LAST_QUEUED_TRANSACTION_END_QUEUE_TIMESTAMP - LAST_QUEUED_TRANSACTION_START_QUEUE_TIMESTAMP 'time_RL_seconds', 
+LAST_APPLIED_TRANSACTION_END_APPLY_TIMESTAMP - LAST_APPLIED_TRANSACTION_START_APPLY_TIMESTAMP 'apply_time_seconds', 
+if(GTID_SUBTRACT(LAST_QUEUED_TRANSACTION, LAST_APPLIED_TRANSACTION) = '','0' , abs(time_to_sec(if(time_to_sec(APPLYING_TRANSACTION_ORIGINAL_COMMIT_TIMESTAMP)=0,0,timediff(APPLYING_TRANSACTION_ORIGINAL_COMMIT_TIMESTAMP,now()))))) `lag_in_seconds` 
+FROM performance_schema.replication_connection_status AS conn_status JOIN performance_schema.replication_applier_status_by_worker AS applier_status 
+ON applier_status.channel_name = conn_status.channel_name WHERE conn_status.service_state = 'ON' ORDER BY lag_in_seconds, lag_in_seconds desc;"
+  metrics:
+    - channel_name:
+        usage: "LABEL"
+        description: ""
```
Comment (BupycHuk) [inline queries-mysqld-group-replication.yml:73]: Why no descriptions here?

File: queries-mysqld-group-replication.yml (hunk 1 above)
Comment (adivinho) [reply queries-mysqld-group-replication.yml]: It has been added descriptions here.

File: queries-mysqld-group-replication.yml (hunk 2 above)
Comment (adivinho) [reply queries-mysqld-group-replication.yml]: Descriptions would be added during the next sprint.
https://jira.percona.com/browse/PMM-6437

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**